### ISSUE TITLE
Custom ERD foreign key relations not showing for other members

### ISSUE THREAD

**Issue by ghost** [dbeaver/dbeaver#13503]
#### System information: 
- Operating system (distribution) and version : Windows 10
- DBeaver version: Community Edition 21.0.5
- Java version "1.8.0_291"
- Additional extensions

#### Connection specification:
- Database name and version : SQL Server 2016
- Driver name
- Do you use tunnels or proxies (SSH, SOCKS, etc)? 

#### Describe the problem you're observing:
I have created a custom ERD by mapping the virtual foreign keys for the tables in my database. However, When I share this custom ERD file with my team, they are unable to view any of the data. 

I then went through the past issues and found this https://github.com/dbeaver/dbeaver/issues/2440. I did as mentioned by one of the user in that message. I updated the data source id for my other users and opened the file in their system. They are now able to see all the files in the custom ERD, however all the foreign key relationships that i created initially are gone. Users are only able to see the tables and the relations are missing. 

Can you please advice what is the best way to share the custom ERD diagrams with other members of the team outside of git framework.

**Comment by Matvey16** [comment]
@KarthikDulam Unfortunately it is impossible for now. We will think about it and see if there is a way to implement a solution for this.

**Comment by ghost** [comment]
@Matvey16 , Thank you for the message. 

Is this something thats available in the Dbeaver enterprise version or with integration with Git? 

If you can suggest some ways that this can be achieved, it will greatly help my cause.

**Comment by Matvey16** [comment]
@KarthikDulam Yes, actually the only possible way to do this is either manually copying the whole project or by making a repository and using the git extension.

**Comment by BryanGuapulema** [comment]
Hi. I have a similar problem. I created my ERD files on a project and pushed them to a repo in Github through Git complement in Dbeaber. But when another member of my team clone this project from github repository it only appears the tables but not the relationships. I am using Bigquery as DB connection and virtual keys for making relationships as long as BQ doesn´t have PK or FK. I tried by coying full folder from one computer to another and it worked but when I restarted the computer the files just dissapear.
